src/mainboard/clevo/adl-p/variants/nv40pz/overridetree.cb

```diff
@@ -1,7 +1,13 @@
 chip soc/intel/alderlake
+	# Default values at boot, to be adjusted by EC at runtime according
+    # to available wall brick capabilities.
 	register "power_limits_config[ADL_P_282_482_28W_CORE]" = "{
 		.tdp_pl1_override = 28,
 		.tdp_pl2_override = 60,
```

Comment:
BTW why NV4x has higher power limits? Does it have better cooling than NS5x?

Reply:
Correct, it has dual fans, NS5x has one fan only.